packages/blade/src/components/Accordion/AccordionItem.tsx

```diff
@@ -71,17 +104,11 @@ const AccordionItem = ({
         <AccordionButton index={_index} icon={icon}>
           {title}
         </AccordionButton>
-        <CollapsibleBody>
-          <BaseBox
-            display="flex"
-            flexDirection="column"
-            gap="spacing.5"
-            marginBottom="spacing.5"
-            marginX="spacing.5"
-          >
-            {_description}
-            {children}
-          </BaseBox>
+        <CollapsibleBody
+          // Just React Native things, need this 100% so collapsed content flows correctly inside Accordion
+          _width={isReactNative() ? '100%' : undefined}
```

Comment:
Wondering if there are enough RN specific things in AccordionItem to split this into native and web files?

Reply:
I think should be fine right now, using RN specific checks at couple of places:
- here for width
- above for adjusting some jsx markup